sci-physics/thepeg/thepeg-2.3.0.ebuild

```
@@ -0,0 +1,108 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools elisp-common java-pkg-opt-2
+
+MY_P=ThePEG-${PV}
+
+DESCRIPTION="Toolkit for High Energy Physics Event Generation"
+HOMEPAGE="http://home.thep.lu.se/ThePEG/"
+
+TEST_URI="https://www.hepforge.org/archive/lhapdf/pdfsets/current"
+SRC_URI="https://www.hepforge.org/archive/thepeg/${MY_P}.tar.bz2
+	test? (
+		hepmc3? (
+			${TEST_URI}/cteq6ll.LHpdf
+			${TEST_URI}/cteq5l.LHgrid
+			${TEST_URI}/GRV98nlo.LHgrid
+			${TEST_URI}/MRST2001nlo.LHgrid )
+	)"
+
+LICENSE="GPL-2"
+SLOT="0/30"
+KEYWORDS="~amd64"
+IUSE="emacs fastjet +hepmc3 lhapdf static-libs test zlib rivet"
+RESTRICT="!test? ( test )"
+
+CDEPEND="
+	sci-libs/gsl:0=
+	emacs? ( >=app-editors/emacs-23.1:* )
+	fastjet? ( sci-physics/fastjet:0= )
+	rivet? ( sci-physics/rivet:3=[hepmc3] )
+	hepmc3? ( sci-physics/hepmc:3= )
+	lhapdf? ( >=sci-physics/lhapdf-6.0:0= )
+	zlib? ( sys-libs/zlib:0= )"
+DEPEND="${CDEPEND}
+	sci-libs/gsl
+	java? ( virtual/jdk:1.8 )
+	test? (
+		sys-process/time
+		dev-libs/boost
+	)"
+RDEPEND="${CDEPEND}
+	java? ( virtual/jre:1.8 )"
```

```suggestion
	java? ( virtual/jre:1.8 )
"
```

Can it be used with newer java? 8 is quite old.

---

I haven't used it with it with java, this is still from the ebuilds in `::gentoo`.